src/tests/Regressions/coreclr/22021/provider.ilproj

```diff
@@ -1,6 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk.IL">
   <PropertyGroup>
     <OutputType>Library</OutputType>
+    <CLRTestKind>BuildOnly</CLRTestKind>
```

Comment:
I think D.B.targets will now automatically set this to `SharedLibrary` - did you hit a problem with that?

Reply:
Thanks for pointing that out, I tried to rebuild the tests without the property and it works fine, in fact I could remove both properties as I believe OutputType is Library by default. Fixed in 15th commit.